kernel/dev/rtc/rtc.cpp

```diff
@@ -1,14 +1,17 @@
 /**
  * @file rtc.cpp
  * @author Keeton Feavel (keetonfeavel@cedarville.edu)
+ * @author Michel (JMallone) Gomes (michels@utfpr.edu.br)
  * @brief
- * @version 0.3
- * @date 2019-11-18
+ * @version 0.4
+ * @date 2021-07-25
  *
  * @copyright Copyright the Panix Contributors (c) 2019
+ * @copyright Copyright 2019, 2020, 2021 mintsuki and contributors.
```

Comment:
i have a doubt about copyright mention in this case. Is right put this ?

Reply:
I would remove the copyright and put a reference instead once you change the code. Move the wikipedia link under the references too.

Reply:
Okey, thanks
I'll do it later